components/clinical/timeline/src/atoms/timeline-time.tsx

```diff
@@ -7,9 +7,33 @@ import {
   TimelineDomainResourceType,
 } from '@ltht-react/types'
 import { formatTime } from '@ltht-react/utils'
+import Icon from "@ltht-react/icon";
+import styled from "@emotion/styled";
 
-const TimelineTime: FC<Props> = ({ domainResource, domainResourceType, ...rest }) => {
+const StyledLeftTimeIcon = styled(Icon)`
+  cursor: pointer;
+  display: inline-block;
+  padding-right: 0.25rem;
+`
+
+const StyledRightTimeIcon = styled(Icon)`
+  cursor: pointer;
+  display: inline-block;
+  padding-left: 0.35rem;
+`
+
+const TimelineTime: FC<Props> = ({ domainResource, domainResourceType, orientation, pointInTimeClickHandler, ...rest }) => {
   if (!domainResource) return <></>
```

Comment:
Not your change but should return `null` rather than `<></>` for better performance (React doesn't have to evaluate `null` jsx values)